src/Migrations/Version20181208160025.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace BalticRobo\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20181208160025 extends AbstractMigration
```

Comment:
It's fine that you use `bin/console doctrine:migration:diff` to create this migration but you should cleanup after. 

Look what I did with `users` migration: https://github.com/balticrobo/website-api/blob/master/src/Migrations/Version20181121164937.php
It extends our very own `AbstractMigration` which checks that database is MySQL so line
```php
$this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```
is not needed anymore.

Your diff changes something on `users` table so if you don't want to change anything in `users` remove it.
```php
$this->addSql('ALTER TABLE users CHANGE roles roles JSON NOT NULL COMMENT \'(DC2Type:json_collection)\', CHANGE created_at created_at INT NOT NULL COMMENT \'(DC2Type:timestamp_immutable)\', CHANGE last_login_at last_login_at INT DEFAULT NULL COMMENT \'(DC2Type:timestamp_immutable)\'');
```
I want to remove all Doctrine comments from database.

Also you keep comments, remove them.